src/main/java/com/google/api/generator/spring/SpringWriter.java

```diff
@@ -179,8 +179,8 @@ private static void writeSpringAdditionalMetadataJson(GapicContext context, JarO
                               + "            \"description\": \"Auto-configure Google Cloud %s components.\",\n"
                               + "            \"defaultValue\": true\n"
                               + "        }",
-                          Utils.springPropertyPrefix(libName, service.name()),
-                          libName + "/" + service.name())));
+                          Utils.getSpringPropertyPrefix(packageName, service.name()),
+                          Utils.getLibName(context))));
```

Comment:
I think we still want `service.name()` some where in the description, as this block is generated per service. So for client lib with multiple services like `vision`, description for enabling each service is different.

Reply:
Oops - thanks for catching this!